README.md

```diff
@@ -1 +1,115 @@
-# Leaderbaord
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do List] <a name="about-project"></a>
+Leader Baoard project is a project that I have created with using Webpack, html,css and javaScript.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://w3cshcool.com/">Webpack,HTML & CSS and JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Used DOM manipulation for implementaion]**
+- **[Used webpack]**
+- **[A wonderfull codeing stracture]**
+- **[Dividing the project into small pecies]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
```

Comment:
- [ ] It would be nice if you make your Live Demo link work because right now it is broken 😨 kindly make it work or remove it from your readme to make your readme descriptive and professional 👍